fusor-ember-cli/app/routes/register-nodes.js

```diff
@@ -85,19 +85,19 @@ export default Ember.Route.extend({
     }).then((result) => {
       controller.set('ports', result.ports);
     }, (error) => {
-      return this.send('error', error, `Unable to load node ports. GET "${url}" failed with status code ${error.jqXHR.status}.`);
+      this.send('error', error, `Unable to load node ports. GET "${url}" failed with status code ${error.jqXHR.status}.`);
     });
   },
 
   loadIntrospectionTasks() {
     let controller = this.get('controller');
     let deploymentId = this.get('controller.deploymentId');
-    this.store.findRecord('deployment', deploymentId, {reload: true}).then(
+    return this.store.findRecord('deployment', deploymentId, {reload: true}).then(
```

Comment:
Another code style q, are you deliberately using the 2 arg `then(=>{}, =>{})` instead of `.then(=>{}).catch(=>{})`? Probably doesn't matter here, but the latter is easier to change for promise chains.